Package.swift

```diff
@@ -32,7 +32,7 @@ let package = Package(
     firebaseDependency(),
     .package(
       url: "https://github.com/grpc/grpc-swift.git",
-      from: "1.19.1" // TODO: Constrain to a range at time of release
+      exact: "1.23.1" 
```

Comment:
We often allow dependency patch version updates like https://github.com/firebase/firebase-ios-sdk/blob/main/Package.swift#L153 depending upon our level of trust of the dependency.

Reply:
Since its the first version, I was going to constrain it to be exact. I'll update it later to 1.23.1 ..< 1.24.0